### ISSUE TITLE
Ensure that elements follow a logical structure independent of styling and positioning

### ISSUE THREAD

**Issue by isaacfallon** [isaacfallon/1-Challenge-Bootcamp#3]
The codebase should have elements that follow a logical structure for readibility and accessibility.
